src/actions/SearchAndFilterActions.js

```diff
@@ -0,0 +1,11 @@
+import {
+    FILTER_CRITERIA_UPDATED
+} from './types';
+
+
+export const checkboxWasClicked = ({filterKey}) => {
+    return{
+        type: FILTER_CRITERIA_UPDATED,
+        payload: {filterKey}
+    }
```

Comment:
Created `SearchAndFilterActions` which will handle any filtering or searching we do. But note that even though these actions have their own actions file, they still route to the `MapReducer` since at the end of the day that's the state set of data that cares about this filtered information.